packages/command-palette/src/commands.tsx

```diff
@@ -308,6 +312,8 @@ export function useCommands() {
 						'Keyword for the Open hosting configuration command',
 						__i18n_text_domain__
 					),
+					'wp cli', // WP-CLI command
+					'wp db', // WP-CLI command
```

Comment:
Shouldn't this be under the `openPHPmyAdmin` command rather than the `openHostingConfiguration` command?